src/chttpd/src/chttpd_httpd_handlers.erl

```diff
@@ -35,6 +35,7 @@ db_handler(<<"_design">>) -> fun chttpd_db:handle_design_req/2;
 db_handler(<<"_partition">>) -> fun chttpd_db:handle_partition_req/2;
 db_handler(<<"_temp_view">>) -> fun chttpd_view:handle_temp_view_req/2;
 db_handler(<<"_changes">>) -> fun chttpd_db:handle_changes_req/2;
+db_handler(<<"_purged_infos">>) -> fun chttpd_db:handle_purged_infos_req/2;
```

Comment:
We don't have a special handler for `_purge` wonder why we need it for `_purged_infos`. Could the regular request handler in chttpd_db be able to handle `_purged_infos`?

Reply:
Great idea, after modification, the code is much cleaner!

Reply:
Perfect. Looks great, @jiahuili430 !